packages/cmrjs/cmr-utils.js

```diff
@@ -379,7 +380,9 @@ function mergeURLs(original, updated, removed = []) {
 async function updateUMMGMetadata(cmrFile, files, distEndpoint, buckets) {
   const newURLs = constructOnlineAccessUrls(files, distEndpoint, buckets);
   const removedURLs = onlineAccessURLsToRemove(files, buckets);
-  const metadataObject = await metadataObjectFromCMRJSONFile(cmrFile.filename);
+  const filename = cmrFile.filename
```

Comment:
Still a bit confused why we're checking the `filename` property?

Reply:
This is one of the instances where, depending on where it's being called from, you're going to have different types of files.

When a granule is moved using the API, the files that make it to here will have `bucket` and `filepath` fields, but no `filename`.  This code is also called as part of the `move-granules` task, which will send it a `filename`.